.vscode/settings.json

```diff
@@ -11,6 +11,7 @@
     "appdirs",
     "camelcase",
     "chokidar",
+    "Codespaces",
```

Comment:
Another question is whether this file affects the VS Code extension itself. May be confusing to include it in the repo if not...

Reply:
Only developer experience.